ebi_eva_common_pyutils/spring_properties.py

```diff
@@ -121,6 +123,95 @@ def get_accessioning_properties(self, *, instance=None, target_assembly=None,  f
+            'config.restartability.allow': 'false',
+            'config.db.read-preference': read_preference,
+
+            'logging.level.embl.ebi.variation.eva': 'DEBUG',
+            'logging.level.org.opencb.opencga': 'DEBUG',
+            'logging.level.org.springframework': 'INFO',
+
+            'spring.main.allow-bean-definition-overriding': 'true',
+            }
+
+    def get_accession_import_properties(self, opencga_path, read_preference='secondaryPreferred'):
+        variants_collection = get_properties_from_xml_file(
+            self.maven_profile, self.private_settings_file)['eva.mongo.collections.variants']
+
+        return self._format(
+            self._common_eva_pipeline_properties(opencga_path, read_preference),
+            {
+                'db.collections.variants.name': variants_collection,
+            },
+        )
+
+    def get_variant_load_properties(self,  project_accession, study_name, output_dir, annotation_dir, stats_dir,
+                                    vep_cache_path, opencga_path, read_preference='secondaryPreferred'):
+        variants_collection = get_properties_from_xml_file(
+            self.maven_profile, self.private_settings_file)['eva.mongo.collections.variants']
+        files_collection = get_properties_from_xml_file(
+            self.maven_profile, self.private_settings_file)['eva.mongo.collections.files']
+        annotation_metadata_collection = get_properties_from_xml_file(
+            self.maven_profile, self.private_settings_file)['eva.mongo.collections.annotation-metadata']
+        annotation_collection = get_properties_from_xml_file(
+            self.maven_profile, self.private_settings_file)['eva.mongo.collections.annotations']
+
+        return self._format(
+            self._common_eva_pipeline_properties(opencga_path, read_preference),
+            {
+            'annotation.overwrite': False,
+            'app.vep.cache.path': vep_cache_path,
+            'app.vep.num-forks': 4,
+            'app.vep.timeout': 500,
+            'config.chunk.size': 200,
+
+            'db.collections.variants.name': variants_collection,
+            'db.collections.files.name': files_collection,
+            'db.collections.annotation-metadata.name': annotation_metadata_collection,
+            'db.collections.annotations.name': annotation_collection,
+
+            'input.study.id': project_accession,
+            'input.study.name': study_name,
+            'input.study.type': 'COLLECTION',
+
+            'output.dir': str(output_dir),
+            'output.dir.annotation': str(annotation_dir),
+            'output.dir.statistics': str(stats_dir),
+
+            'spring.jpa.database-platform': 'org.hibernate.dialect.PostgreSQL9Dialect',
+            'spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation': True,
+            'spring.jpa.properties.hibernate.temp.use_jdbc_metadata_defaults': False,
+            'statistics.skip': False
+        },
```

Comment:
This [comment](https://github.com/EBIvariation/eva-common-pyutils/pull/36/files#r1180212580) applies here too...

Reply:
reformatted the code to reuse common properties